services/queue/schemas/v1/tasks-response.yml

```
@@ -0,0 +1,32 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Task Definition Response"
+description: |
+  Definitions of tasks that can be scheduled
```

```suggestion
  Definitions of tasks
```

Same here, endpoint will return any tasks